index.html

```diff
@@ -4,11 +4,13 @@
     <meta charset="utf-8">
     <title></title>
     <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.4.0/Chart.min.js"></script>
   </head>
   <body>
     <div id="pictures">
 
     </div>
+    <canvas id="productChoices"></canvas>
```

Comment:
This is super nitpicky, but in general, use camelCase for javascript variables, and kebab-case for ids and classes.